src/Symfony/Component/HttpKernel/Debug/FileLinkFormatter.php

```diff
@@ -24,9 +23,9 @@ class FileLinkFormatter implements \Serializable
     private $fileLinkFormat;
     private $requestStack;
     private $baseDir;
-    private $urlFormat;
+    private $queryString;
 
-    public function __construct($fileLinkFormat = null, RequestStack $requestStack = null, $baseDir = null, $urlFormat = null)
+    public function __construct($fileLinkFormat = null, RouterInterface $router = null, $baseDir = null, $queryString = null)
```

Comment:
Changing the signature is a BC break (it might break SilexWebProfiler btw)

Reply:
If I'm right, we don't care about BC breaks in debug tools like this one.